care/facility/api/viewsets/facility_users.py

```diff
@@ -49,5 +50,5 @@ def get_queryset(self):
                     queryset=UserSkill.objects.filter(skill__deleted=False),
                 ),
             )
-        except Facility.DoesNotExist:
-            raise ValidationError({"Facility": "Facility not found"})
+        except Exception as e:
```

Comment:
Why is the exception being replaced with a generic one? This seems like going backwards